geomesa-hbase/geomesa-hbase-dist/src/main/assemblies/component.xml

```diff
@@ -106,6 +106,12 @@
             <destName>README-convert.md</destName>
             <outputDirectory>docs</outputDirectory>
         </file>
+        <!-- geomesa-site.xml.template -->
+        <file>
+            <source>../../../geomesa-utils/src/main/resources/org/locationtech/geomesa/geomesa-site.xml.template</source>
```

Comment:
agreed, but i think it's the only thing maven supports that still works if you don't build from the top-level dir...